### DISCUSSION TITLE
Copilot on IntelliJ should not ruin intellisense

### DISCUSSION THREAD

**Discussion by villesau** [community/community#44806] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot eagerly shows it's guesses on what to write. However, they are _guesses_ while IntelliJ intellisense shows real suggestions on what is possible. This ruins the experience on IntelliJ as the intellisense suggestions should have priority over the copilot. I hope you get this fixed as I would like to benefit of copilots guesses too.

**Comment by iamcjwilliams** [comment]
@villesau I'll be sure to notify our copilot team, thank you for your feedback! 🚀
